backend/program/settings/models.py

```diff
@@ -62,7 +64,7 @@ def check_update_interval(cls, v):
 class PlexLibraryModel(Updatable):
     update_interval: int = 120
     token: str = ""
-    url: str = "http://localhost:32400"
+    url: str = "http://plex:32400" if is_elfhosted else "http://localhost:32400"
```

Comment:
Instead of having these scattered around should they be gathered into one method that changes the variables on startup or an override of the models. This way it wont get so clustered

Reply:
since majority of people should be fine with using container name, we can probably just default to that initially? and for those that want to run it outside of the container can just change it to `localhost` instead..

question here is.. who's our target audience?